virttest/qemu_devices/qdevices.py

```diff
@@ -1009,8 +1009,6 @@ def __init__(self,
                  child_bus=None):
         backend = params.get("backend", "socket")
         self.verify_supported_backend(backend)
-        options = self.get_supported_options(backend)
-        params = params.copy_from_keys(options)
         params = self.format_params(params)
```

Comment:
After remove `params = params.copy_from_keys(options)`,  `params=None` here, so it is meaningless for `format_params`.

Reply:
@yanan-fu Actually they are independent, `format_params()` translate params before `yes` `no` `Ture` `False`, which is still in required here.